targets/TARGET_NORDIC/TARGET_NRF5/TARGET_MCU_NRF52840/sdk/sdk_config.h

```diff
@@ -1311,6 +1311,8 @@
 #endif //PWM_CONFIG_LOG_ENABLED
 // </e>
 
+#define PWM_COUNT (PWM0_ENABLED + PWM1_ENABLED + PWM2_ENABLED + PWM3_ENABLED)
```

Comment:
Please remove this line. This definition is already in code: https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_NORDIC/TARGET_NRF5/TARGET_SDK13/device/nrf52840_peripherals.h#L221